_posts/2018/2018-06-18-bridging-the-gap-between-literate-programming-and-teaching-modules.markdown

```diff
@@ -0,0 +1,94 @@
+---
+layout: "post"
+title: "Bridging the gap between literate programming and teaching modules"
```

Comment:
A potential variant for the title: leveraging  literate computing technology for teaching: a template web site for jupyter-based courses